Vagrantfile

```diff
@@ -82,8 +82,8 @@ Vagrant.configure(2) do |config|
 
     vb.customize ['modifyvm', :id, '--paravirtprovider', 'minimal']
 
-    default_machine_folder = `VBoxManage list systemproperties | grep "Default machine folder"`
-    vb_machine_folder = default_machine_folder.split(':')[1].strip()
+    default_machine_folder = `(which VBoxManage 1> /dev/null 2> /dev/null && (VBoxManage list systemproperties | grep "Default machine folder" | sed "s/Default machine folder://")) || echo "/tmp"`
+    vb_machine_folder = default_machine_folder.strip()
```

Comment:
From chat ... Better to check early in the section if VBoxManage exists, and if not, return.

Reply:
Good error message too, like "Virtualbox provider requires installation of VBoxManage" etc. That way somebody who forgot to install the prequisites of the provider gets a reminder.

Reply:
I'm going with a different approach. Let's store the VirtualBox disks under `<project_root>/.vagrant/disks`.